DigitalLearningSolutions.Data/Services/CourseDelegatesDownloadFileService.cs

```diff
@@ -65,11 +62,9 @@ public byte[] GetCourseDelegateDownloadFileForCourse(int customisationId, int ce
         private void PopulateCourseDelegatesSheetForCourse(
             IXLWorkbook workbook,
             int customisationId,
-            int centreId,
-            int? adminCategoryId
+            int centreId
         )
         {
-            var course = courseService.GetCourseDetailsFilteredByCategory(customisationId, centreId, adminCategoryId)!;

```

Comment:
I'm going to guess that this is what was causing this bug? Would the service not get allCentre courses, so things would break further down the line if this was actually null? The developer checks on the MR state that unit tests were written, but I don't think any unit test can be, and it doesn't look like any have been.

Reply:
Yeah this method doesn't get all centre courses, and wasn't necessary any more anyway as I changed the way it was being used previously so that it wasn't doing anything useful.